modules/vaos/app/services/eps/appointment_service.rb

```diff
@@ -81,38 +89,21 @@ def create_draft_appointment(referral_id:)
     # @return OpenStruct response from EPS submit appointment endpoint
     #
     def submit_appointment(appointment_id, params = {})
-      raise ArgumentError, 'appointment_id is required and cannot be blank' if appointment_id.blank?
-      raise ArgumentError, 'Email is required' if user.email.blank?
-
-      required_params = %i[network_id provider_service_id slot_ids referral_number]
-      missing_params = required_params - params.keys
-
-      raise ArgumentError, "Missing required parameters: #{missing_params.join(', ')}" if missing_params.any?
-
+      validate_submit_params!(appointment_id, params)
```

Comment:
with the rescue block this method became too long for rubocop length rule so it had to be broken out